stdlib/stdlib.mli

```diff
@@ -815,7 +815,7 @@ external snd : 'a * 'b -> 'b = "%field1"
 *)
 
 val ( @ ) : 'a list -> 'a list -> 'a list
-(** List concatenation.  Not tail-recursive (length of the first argument).
+(** List concatenation.
```

Comment:
Same comment here on the doc string. This could be:

```
[l0 @ l1] concatenates [l1] to [l0]. Same function as {!List.append}.
```

Reply:
Done; thanks!